lib/usage.txt

```diff
@@ -15,6 +15,12 @@ Install a generator:
   $ npm install generator-angular
   $ yo angular --help
 
+Run local generators:
+  
+  Additionally, you can also run local generators without installing via the npm.
```

Comment:
There seems to be an unneeded article: »via _**the**_ npm«. 
Please remove.

Reply:
Okay

Reply:
@mischah There :)